mslib/msui/constants.py

```diff
@@ -34,11 +34,12 @@
 # ToDo refactor to generic functions, keep only constants
 HOME = os.path.expanduser(f"~{os.path.sep}")
 MSUI_CONFIG_PATH = os.getenv("MSUI_CONFIG_PATH", os.path.join(HOME, ".config", "msui"))
+_fs = None
```

Comment:
This doesn't solve the issue. Now it is possible that `.makedirs` is called on `None`.

Reply:
Please remove this `_fs = None` line, it should not be necessary.